addons/portal/wizard/portal_wizard.py

```diff
@@ -88,6 +88,24 @@ class PortalWizardUser(models.TransientModel):
     login_date = fields.Datetime(related='user_id.login_date', string='Latest Authentication')
     is_portal = fields.Boolean('Is Portal', compute='_compute_group_details')
     is_internal = fields.Boolean('Is Internal', compute='_compute_group_details')
+    email_state = fields.Selection([
+        ('ok', 'Valid'),
+        ('ko', 'Invalid'),
+        ('exist', 'Already Registered')],
+        string='Status', compute='_compute_email_state', default='ok')
+
+    @api.depends('email')
+    def _compute_email_state(self):
+        for portal_wizard_user in self:
+            email_normalized = email_normalize(portal_wizard_user.email)
+            if not email_normalized:
+                portal_wizard_user.email_state = 'ko'
+            else:
+                existing_identical_users_count = self.env['res.users'].sudo().with_context(active_test=False).search_count([
```

Comment:
Note that internally, as we have a lot of users, using a search_count in a loop is probably costly ... Isn't it possible to fetch all normalized emails, search users related to those emails (exclude False/void/ ...), and update accordingly afterwards ?